ui/src/components/home/NewHereSection.js

```diff
@@ -45,112 +47,122 @@ const NewHereSection = () => {
   };
 
   const resetForm = () => {
-    setName("");
-    setEmail("");
-    setPhoneNumber("");
-    setLifestage("");
-    setNotes("");
-    setPhoneNumber("");
-    setLifestage("");
+    setName('');
+    setEmail('');
+    setPhoneNumber('');
+    setLifestage('');
+    setNotes('');
+    setPhoneNumber('');
+    setLifestage('');
     setIsBot(true);
     setSubmitted(false);
   };
 
   return (
     <>
       <Flex
-        w="full"
-        h="100vh"
-        bgImage={`linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ),url(${
-          process.env.PUBLIC_URL + "/images/home/connect.png"
-        })`}
-        bgSize="cover"
-        bgPosition="center center"
+        w={'full'}
+        h={['110vh', '80vh']}
         justify="center"
+        background="#F9F9F9"
+        style={{ backdropFilter: 'blur(7px)' }}
       >
-        <Flex
-          w="full"
-          h="100%"
-          justify="center"
-          background='linear-gradient(90deg, rgba(6, 40, 163, 0.6) 0%, rgba(196, 241, 249, 0.6) 100%);"    '
-          style={{ backdropFilter: "blur(7px)" }}
+        <BackgroundElements />
+        <Container
+          maxW="container.xl"
+          justifyContent="center"
+          alignItems="center"
+          pt={['35', '55']}
         >
-          <Container maxW="container.lg" justifyContent="center" display="flex">
-            <VStack color="white" justify="center" spacing={5}>
-              <Heading fontSize={["1.3em", "3em"]}>
-                New here? Connect with us
+          <Stack
+            direction={['column', 'row']}
+            color="white"
+            justify={'center'}
+            alignItems={['left', 'center']}
+            spacing={['5', '50']}
+          >
+            <VStack alignItems={'left'} spacing={0}>
+              <Heading fontSize={['2em', '3em']} color="#505050">
+                New here?
               </Heading>
+              <Heading fontSize={['2em', '3em']} color="#0628A3">
+                Connect with us
+              </Heading>
+            </VStack>
+            <Box
+              h={['75vh', '65vh']}
```

Comment:
![Screenshot 2022-08-19 at 16 52 37](https://user-images.githubusercontent.com/33679483/185582609-24c88412-c60b-41cd-9117-1b4127317eee.png)

this doesnt look right

Reply:
i think you have to make the box wider and also make the fields inside wider + taller, including the buttons. refer to figma